ui/v2/src/components/select/FilterMultiSelect.tsx

```diff
@@ -106,12 +107,19 @@ export const FilterMultiSelect: React.FunctionComponent<IProps> = (props: IProps
         translateData = () => { let perfData = data as GQL.AllPerformersForFilterQuery; setItems(!!perfData && !!perfData.allPerformers ? perfData.allPerformers : []); };
         break;
       }
-      case "studios": {
+     /*  case "studios": {
```

Comment:
What is the purpose of commenting this out? If it's superfluous it should probably just be removed altogether.

Reply:
I will revise it.